io_directory_example/settings/_test_config2.yml

```diff
@@ -0,0 +1,56 @@
+# this config file is only used for testing, do not directly copy for use
```

Comment:
Is it possible to consolidate "_test_config1.yml" and "_test_config2.yml" into the corresponding *.yml file for "lens_system3"? The idea is one config file per system within the data folder. The io_directly_example directory is not only used for testing, but it is also an example for users on how to structure their data and config files.

Reply:
I am renaming test_config2.yml to lens_system3.yml.  test_config1.yml has the same system as lens_system_1, and has been in the code before me, should I try to consolidate it?

Reply:
Ok, I see that, you can leave _test_config1.yml then, but please consolidate _test_config2.yml if possible. But, if it becomes too complex to do that, then you can leave that as a separate yml file too.